website/docs/markdown-features.mdx

```diff
@@ -1060,3 +1060,26 @@ import DocusaurusSvg from './docusaurus.svg';
 import DocusaurusSvg from '@site/static/img/docusaurus.svg';
 
 <DocusaurusSvg />
+
+### Themed Image
+
+Docusaurus also offers support for the themed images. It is useful if you need to switch the whole image based on the current theme.
+
+```js
+import ThemedImage from '@theme/ThemedImage';
+
+<ThemedImage
+  alt="Docusaurus themed image"
+  lightSrc={useBaseUrl('img/docusaurus_light.svg')}
+  darkSrc={useBaseUrl('img/docusaurus_dark.svg')}
+/>;
+```
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+
+<ThemedImage
+  alt="Docusaurus themed image"
+  lightSrc={useBaseUrl('img/docusaurus_keytar.svg')}
```

Comment:
Why don't we just pass the path as regular string? Accordingly, inside the `ThemedImage` component, apply `useBaseUrl` to all src paths? Then users would not import `useBaseUrl` in their MDX doc.

Reply:
@lex111 totally agree with that. BTW I imagined we could even have a Docusaurus image component just for this usecase

Reply:
I'm not sure if processing the base URL inside the component is a good idea. It can blur the functionality and cause a confusion (why `img` need useBaseURL when `ThemeImage` do not need it? The goal was to create simple helper not the robust, alternative image component.

And again, I have just looked at the way `img` tag is used on the Docusuaurus V2 website - https://github.com/facebook/docusaurus/blob/master/website/src/pages/index.js#L77. 

Let me remind you that is  just a simple helper.

Reply:
My long-term goal is to simply deprecate useBaseUrl(). But this can probably be done later, so if we don't apply it in the component today, it's fine